sdk/src/test/scala/com/horizen/SidechainStateTest.scala

```diff
@@ -52,7 +52,9 @@ class SidechainStateTest
 
   val params = MainNetParams()
 
-
+  val closedForge = false
```

Comment:
add the tests for forgers validation in the `SidechainStateTest` and `SidechainStateIntegrationTest` classes.

Reply:
Done